src/App.js

```diff
@@ -159,7 +169,6 @@ function App() {
                   secondary="Secondary text"
                 />
                 <ArrowListItem primary="Just me and arrow" />
-                <Button text="Show a toast" onClick={() => { showToast(toastValues[Math.round(Math.random() * 2)], 5000) }} />
```

Comment:
I forgot to include this button in the merge. Would be great if you could fix this @DanielBrierton .